system/jlib/jptree.cpp

```diff
@@ -3038,51 +3037,56 @@ void PTree::serialize(MemoryBuffer &tgt)
 
 void PTree::deserialize(MemoryBuffer &src)
```

Comment:
deserialize also needs a context parameter - which would be created in createPTree()